ledger_device_ui_sdk/Cargo.toml

```diff
@@ -7,8 +7,10 @@ license.workspace = true
 description = "Ledger devices abstractions for displaying text, icons, menus and other common gadgets to the screen"
 
 [dependencies]
-ledger_secure_sdk_sys = "1.0.0"
-include_gif = "1.0.0"
+ledger_device_sdk = { git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git", branch = "add-ui-gadgets" }
+ledger_secure_sdk_sys = { git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git", branch = "add-ui-gadgets" }
+include_gif = { git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git", branch = "add-ui-gadgets" }
+numtoa = "0.2.4"
```

Comment:
dependencies should be udpated to crates.io :
```
ledger_device_sdk = "1.0.0"
ledger_secure_sdk_sys = "1.0.0"
include_gif = "1.0.0"
```
and new `ledger_device_ui_sdk` should be published on crates.io

I still think that making `ledger_device_ui_sdk` depending on `ledger_device_sdk` is weird and break the planned architecture (even if I understand the needs). That's why I think that later, this crate shall be integrated into `ledger_device_sdk` (as a `ui` module for instance).

Reply:
Done